app/controllers/v1/decision_review_evidences_controller.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
```

Comment:
This is an exact copy of `v0/decision_review_evidences_controller.rb` — we just wanted to switch everything over to v1 so it's a cleaner break from using v0

Reply:
Will the FE need to be updated before this is merged?

Reply:
nah, this can get merged because we haven't deleted v0 for that controller yet. once this is merged, we can switch the FE to use this v1, and then we'll follow up and remove the v0 🤠